x/projects/client/cli/tx_add_keys.go

```diff
@@ -15,18 +15,18 @@ import (
 
 var _ = strconv.Itoa(0)
 
-func CmdAddProjectKeys() *cobra.Command {
+func CmdAddKeys() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "add-project-keys [project-id] [optional: project-keys-file-path]",
+		Use:   "add-keys [project-id] [optional: project-keys-file-path]",
 		Short: "Add developer/admin keys to an existing project",
-		Long: `The add-project-keys command allows the project admin to add new project keys (admin/developer) to the project.
+		Long: `The add-keys command allows the project admin to add new project keys (admin/developer) to the project.
 		To add the keys you can optionally provide a YAML file of the new project keys (see example in cookbook/project/example_project_keys.yml).
 		Note that in project keys, to define the key type, you should follow the enum described in the top of example_project_keys.yml.
 		Another way to add keys is with the --admin and --developer flags. Note that the developer must come with its VRF key (see example)`,
 		Example: `required flags: --from <admin-key> (the project's subscription address is also considered admin)
 				  
-		lavad tx subscription add-project-keys [project-id] [project-keys-file-path] --from <admin-key>
-		lavad tx subscription add-project-keys [project-id] --admin <other-admin-key> --admin <another-admin-key> --developer <developer-key>:<developer-vrfpk> --from <admin-key>`,
+		lavad tx project add-keys [project-id] [project-keys-file-path] --from <admin-key>
+		lavad tx project add-keys [project-id] --admin <other-admin-key> --admin <another-admin-key> --developer <developer-key>:<developer-vrfpk> --from <admin-key>`,
```

Comment:
in the usage of the developer key flag, there is a use of vrfpk. Is it still relevant? if not, the comment and the code that handles the developer flag should be changed

Reply:
vrf removal is on another task :)